Source/Core/VideoCommon/RenderBase.h

```diff
@@ -135,6 +136,7 @@ class Renderer
 	virtual int GetMaxTextureSize() = 0;
 
 	static Common::Event s_screenshotCompleted;
+	
```

Comment:
damn sorry about those.  I'm just not used to having it, you know, actually matter at all what my white space looks like.